thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -347,29 +326,18 @@ export default Controller.extend({
    * @param {Object} newId - conditional param to help us tag any function that was "just added"
    * @return {RSVP.Promise} A new list of functions (alerts)
    */
-  prepareFunctions(configGroup, newId = 0) {
-    const newFunctionList = [];
+  prepareFunctions(configGroup) {
```

Comment:
Looks like this is the same code as https://github.com/linkedin/pinot/pull/2545/files#diff-c4e1ed83815b62ca0daad6dbd8dbe9d8R234

Any way we can share this function in both places?

Reply:
This is a good candidate to move into shared services for self-serve. Added a refactor note.